Add a few more questions and feature comparisons to lazer migrations page (#8476)
ppy/osu-wiki: wiki/Help_centre/Upgrading_to_lazer/en.md

| Profile statistics | ✅ | ✅ |
| Medals | ✅ | ❌ |
| Performance points | ✅ | 🟠[^score-reset-isolated] |
| Real-time chat | 🟠[^stable-chat] | ✅ |
| Wiki / News / Changelog / Rankings | ❌ | ✅[^online-content] |
| User profiles | ❌ | ✅ |
| Beatmap listing | 🟠[^direct-supporter] | ✅ |
| Unlimited multiplayer room size | ❌[^multi-room-max] | ✅ |
| Multiplayer spectating | ❌ | ✅ |

> | Countdown timers | 🟠[^countdown-timers-stable] | ✅ [^countdown-timers-lazer] |

Reviewer: ```suggestion
| Countdown timers | 🟠[^countdown-timers-stable] | ✅[^countdown-timers-lazer] |
```

### Other

#### Why is it called "lazer"?

What is sharper than cutting–edge?

#### Why is it taking so long to become the "main" release?

While osu! may seem like a simple game, there are hundreds on hundreds of features and systems that users have come to rely on. Depending on who you ask, lazer may have been in a fully playable state for years now, or it may be missing countless features.

> Another area which has taken a huge amount of effort is historical preservation – making sure that beatmaps behave exactly as they should, including edge cases that weren't originally planned for. osu! is a vibrant ecosystem and users have taken liberty to extend the game far beyond its planned extents, and we are trying our best to embrace and support this going forward.

Reviewer: em dash, not en dash

```suggestion
Another area which has taken a huge amount of effort is historical preservation — making sure that beatmaps behave exactly as they should, including edge cases that weren't originally planned for. osu! is a vibrant ecosystem and users have taken liberty to extend the game far beyond its planned extents, and we are trying our best to embrace and support this going forward.
```

> Finally, unlike the last iteration, we are putting in the time and diligence to ensure the code base will serve us well into the future. We have done the groundwork to allow new features to come online at blazing speed going forward. This will include new UI components, new ways to skin the game, new multiplayer systems and let's not forget the ability to load and play all your existing beatmaps on completely new game modes (aka rulesets)!

Reviewer: ```suggestion
Finally, unlike the last iteration, we are putting in the time and diligence to ensure the code base will serve us well into the future. We have done the groundwork to allow new features to come online at blazing speed going forward. This will include new UI components, new ways to skin the game, new multiplayer systems and let's not forget the ability to load and play all your existing beatmaps on completely new game modes (a.k.a. rulesets)!
```

[^all-files]: All files.
[^editor-precise-rotation]: Missing precise angle rotation.
[^incompatibilities]: Some editor features will cause beatmaps to play incorrectly in stable — will be fixed soon.
[^stable-chat]: Messages can take up to 15 seconds to arrive.
[^countdown-timers-stable]: Set a countdown using a command, no automatic start.
[^countdown-timers-lazer]: Set a countdown from the game UI to automatically start the match.

> [^queue-modes]: Turn on to allow anyone in a lobby to queue new beatmaps, aka "host rotate"

Reviewer: ```suggestion
[^queue-modes]: Turn on to allow anyone in a lobby to queue new beatmaps, a.k.a. "host rotate".
```
